src/main/java/com/github/maxopoly/kira/api/input/packets/RunIngameCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.github.maxopoly.kira.api.input.packets;
+
+import java.util.regex.Pattern;
+
+import org.json.JSONObject;
+
+import com.github.maxopoly.kira.KiraMain;
+import com.github.maxopoly.kira.api.input.APIInput;
+import com.github.maxopoly.kira.api.input.APISupplier;
+import com.github.maxopoly.kira.api.sessions.APIIngameCommandSession;
+
+public class RunIngameCommand extends APIInput {
+
+    private String commandRegex = "[a-zA-Z0-9_\\- !?\\.]+";
+
+    public RunIngameCommand() {
+        super("in-game");
+    }
+
+    @Override
+    public void handle(JSONObject argument, APISupplier supplier) {
+        
```

Comment:
Can you remove all these empty lines inbetween normal logic, they are awful style. Separating blocks of logic is fine, this is not.